README.md

```diff
@@ -15,9 +15,15 @@ This app listens on the Item kinesis stream to identify Item updates that *look
 ## Setup
 ### Installation
 
+1. Open a shell into the docker container:
+`docker run -it --rm -v "$PWD":/var/task lambci/lambda:build-ruby2.5 bash`
```

Comment:
I think it looks good. But we do repeat the line 73. Maybe there's a way to make it better structure?